bigbluebutton-html5/private/locales/en.json

```diff
@@ -172,18 +172,18 @@
     "app.actionsBar.emojiMenu.statusTriggerLabel": "Status",
     "app.actionsBar.emojiMenu.awayLabel": "Away",
     "app.actionsBar.emojiMenu.awayDesc": "Change your status to away",
-    "app.actionsBar.emojiMenu.raiseLabel": "Raise",
-    "app.actionsBar.emojiMenu.raiseDesc": "Raise your hand to ask a question",
-    "app.actionsBar.emojiMenu.undecidedLabel": "Undecided",
-    "app.actionsBar.emojiMenu.undecidedDesc": "Change your status to undecided",
+    "app.actionsBar.emojiMenu.raiseHandLabel": "Raise",
+    "app.actionsBar.emojiMenu.raiseHandDesc": "Raise your hand to ask a question",
+    "app.actionsBar.emojiMenu.neutralLabel": "Undecided",
```

Comment:
undecidedLabel

Reply:
I think you missed the point where he is creating it dynamically and following the values sended to akka-apps. If you want to have a "undecided" status, you send "neutral".

Reply:
imo is not worth to map the locales to something else than what akka expects, that only adds more logic mapping things like neutral -> undecided or clear -> none

Reply:
Oh!
Alright, I was looking at the names of the icons and not to the akka names.